### DISCUSSION TITLE
In 1810, New York overtook what city as America's most populated city?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#225] [Galaxy Trivia] [answered]
**Hint:** cream cheese

**Comment by narayan954** [comment, marked as answer]
Philadelphia

**Reply by ParanoidUser** [reply]
You nailed it! Congratulations on your well deserved [Galaxy Brain](https://github.com/narayan954?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 4 answers), and don't hesitate to post your own questions to the community. 😊🎉

**Reply by narayan954** [reply]
Thanks a lot :), will definitely contribute to the community as well.
